--- .htaccess   2014-08-05 09:27:11.462044910 +1200
+++ .htaccess2  2014-08-05 09:27:33.746044023 +1200
@@ -7,11 +7,11 @@

 # client-side URI mapping
 # if you are running in a top-level domain, you should use this:
-# RewriteBase /
-# ErrorDocument 404 /404.php
+RewriteBase /
+ErrorDocument 404 /404.php
 # if you are running in a subfolder, you should instead use this:
-RewriteBase /clerk
-ErrorDocument 404 /clerk/404.php
+# RewriteBase /clerk
+# ErrorDocument 404 /clerk/404.php

 # Provide human-readable URLs for some common paths; otherwise
 # all scripts are accessible through the final rule below.
